### DISCUSSION TITLE
hexo s  ,that's ok .  but after  hexo d  to github.  error:404

### DISCUSSION THREAD

**Discussion by 123321jobs** [hexojs/hexo#4978] [Question / Answer] [answered]
hello,
      this is my url [:](https://github.com/123321jobs/boke.github.io.git)

      thanks! 
      have a good day!

**Comment by stevenjoezhang** [comment, marked as answer]
Please rename your repository from `boke.github.io` to `123321jobs.github.io`

**Reply by 123321jobs** [reply]
> Please rename your repository from `boke.github.io` to `123321jobs.github.io`
Hi, thank you!  I have a try at now!

**Reply by 123321jobs** [reply]
> Please rename your repository from `boke.github.io` to `123321jobs.github.io`
Wow, thank you, I rename the repo to show it! I love you so much!

**Reply by 123321jobs** [reply]
> Please rename your repository from `boke.github.io` to `123321jobs.github.io`
But I just changed back to the previous name, and the blog can also be displayed, which is a bit strange. It may take some time to display normally after deployment.

**Comment by 123321jobs** [comment]
hi, thank you! I have a try at now!
